planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/src/scene_module/detection_area/manager.cpp

```diff
@@ -59,21 +60,25 @@ DetectionAreaModuleManager::DetectionAreaModuleManager(rclcpp::Node & node)
 {
   const std::string ns(getModuleName());
   planner_param_.stop_margin = node.declare_parameter(ns + "/stop_margin", 0.0);
+  planner_param_.use_dead_line = node.declare_parameter(ns + "/use_dead_line", false);
```

Comment:
Fix parameter delimiter.(Replace / with . as below.) The same is true for other modules.
https://github.com/tier4/AutowareArchitectureProposal.iv/blob/b0ba4f052ed836f5bc69e87d7f3ca83fec37f8e2/planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/src/node.cpp#L96

Reply:
It's Inheritance from reclcpp so either is ok. v0.5.0 uses this style.
But I will fix it.

Reply:
fixed at fbe898a